asistente_ladm_col/gui/change_detection/dlg_change_detection_settings.py

```diff
@@ -0,0 +1,284 @@
+        self.set_init_db_config()
+
+        for mode, label_mode in self.CHANGE_DETECTIONS_MODES.items():
+            self.cbo_change_detection_modes.addItem(label_mode, mode)
+
+        self.radio_button_other_db.setChecked(True)
+        self.radio_button_same_db.setEnabled(True)
+        if not self._db_collected.supplies_model_exists():
+            self.radio_button_same_db.setEnabled(False)
+
+        self.radio_button_same_db.toggled.connect(self.update_supplies_db_options)
+        self.update_supplies_db_options()
+
+        self.btn_collected_db.clicked.connect(self.show_settings_collected_db)
+        self.btn_supplies_db.clicked.connect(self.show_settings_supplies_db)
+
+        # Set connections
+        self.buttonBox.accepted.disconnect()
+        self.buttonBox.accepted.connect(self.accepted)
+        self.buttonBox.helpRequested.connect(self.show_help)
+
+        self.bar = QgsMessageBar()
+        self.bar.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Fixed)
+        self.layout().addWidget(self.bar, 0, 0, Qt.AlignTop)
+
+        self.update_connection_info(COLLECTED_DB_SOURCE)
+        self.update_connection_info(SUPPLIES_DB_SOURCE)
+
+    def set_init_db_config(self):
+        dlg_collected_config = SettingsDialog(qgis_utils=self.qgis_utils, conn_manager=self.conn_manager, db_source=COLLECTED_DB_SOURCE)
+        dlg_supplies_config = SettingsDialog(qgis_utils=self.qgis_utils, conn_manager=self.conn_manager, db_source=SUPPLIES_DB_SOURCE)
+
+        self._init_db_collected_active_mode = dlg_collected_config.cbo_db_source.itemData(dlg_collected_config.cbo_db_source.currentIndex())
+        self._init_db_supplies_active_mode = dlg_supplies_config.cbo_db_source.itemData(dlg_supplies_config.cbo_db_source.currentIndex())
+
+        for id_db, db_factory in dlg_collected_config._lst_db.items():
+            dict_conn = db_factory.get_parameters_conn(COLLECTED_DB_SOURCE)
+            self._init_db_collected_dict_config[id_db] = dict_conn
+
+        for id_db, db_factory in dlg_supplies_config._lst_db.items():
+            dict_conn = db_factory.get_parameters_conn(SUPPLIES_DB_SOURCE)
+            self._init_db_supplies_dict_config[id_db] = dict_conn
+
+    def update_supplies_db_options(self):
+        if self.radio_button_same_db.isChecked():
+            self.btn_supplies_db.setEnabled(False)
+        else:
+            self.btn_supplies_db.setEnabled(True)
+        self.update_connection_info(SUPPLIES_DB_SOURCE)
+
+    def show_settings_collected_db(self):
+        tab_pages_list = [SETTINGS_CONNECTION_TAB_INDEX]
+        dlg = SettingsDialog(qgis_utils=self.qgis_utils, conn_manager=self.conn_manager, tab_pages_list=tab_pages_list)
+        dlg.set_required_models([LADMNames.OPERATION_MODEL_PREFIX])
+
+        # Connect signals (DBUtils, QgisUtils)
+        dlg.db_connection_changed.connect(self.db_connection_changed)
+        dlg.db_connection_changed.connect(self.qgis_utils.cache_layers_and_relations)
```

Comment:
Change it to the new way (using _schedule_refresh_cache_layers_and_relations).

Reply:
Done. Thanks

Reply:
For the future, please specify in which commit the comment was solved.

E.g., Solved in https://github.com/AgenciaImplementacion/Asistente-LADM_COL/pull/284/commits/a17236c930d8e33e688a9ed896adc316c397ef3e